examples/sort_example.c

```diff
@@ -5,43 +5,47 @@
 
 #include "../sort.h"
 
-#define SIZE (50000)
+#define SIZE (5 * 1000000)
+double arr[SIZE];
+double arr2[SIZE];
+
 static inline bool cmp(const void *a, const void *b)
 {
-    return *(int *)a < *(int *)b;
+    return *(double *)a < *(double *)b;
```

Comment:
Is there any reason why double is chosen over long?

Reply:
because datatype being sorted in this version of the example is double.